pytorch_tabnet/tab_model.py

```diff
@@ -242,6 +242,12 @@ def fit(self, X_train, y_train, X_valid=None, y_valid=None, loss_fn=None,
             else:
                 self.patience_counter += 1
 
+            if self.scheduler is not None:
+				if self.scheduler_fn == torch.optim.lr_scheduler.ReduceLROnPlateau:
```

Comment:
You have an indentation problem here I think.

More over, I think this a bit too specific for ReduceLROnPlateau... but at least could you make the if statement use isinstance instead of == ?